adoc/MAIN.release-notes.adoc

```diff
@@ -41,15 +41,15 @@ Add CLI etcdctl to provide access to etcd container.
 
 In order to seamlessly switch to {tf} 0.12 you need to make sure that:
 
-* all files follow the new syntax for the HashiCorp Configuration Language included in {tf} 0.12;
-* all boolean values are `true` or `false` and *not* 0 or 1;
-* all variables are explicitly declared;
-* all dependencies are explicitly declared to reach the expected behavior.
+* All files follow the new syntax for the link:https://github.com/hashicorp/hcl[HashiCorp Configuration Language] included in {tf} 0.12
+* All boolean values are `true` or `false` and *not* 0 or 1
+* All variables are explicitly declared
+* All dependencies are explicitly declared to reach the expected behavior
 
 ==== Recommended Procedure
 
 If you can tear down your existing cluster, do delete your cluster
-before upgrading to {tf} 0.12. After that follow our documentation to create a new cluster.
+before upgrading to {tf} 0.12. After that, follow our documentation to create a new cluster.
 That will lead to the cleanest upgrade result.
```

Comment:
Oh wow, this looks a bit extreme. Is this what we recommend?

Reply:
Don't know, that is the contribution I got from @evrardjp if this is not the intended way to upgrade then please let me know what is. I can't know.

Reply:
This is related to openstack only. Because of a `user_data` change all machines are detected as new resources. The recommendation is to ignore the `user_data` field as documented in https://susedoc.github.io/caasp-release-notes/master/single-html/release-notes/#_extra_operations_for_in_place_upgrade_of_openstack_terraform_files.

cc/ @evrardjp, in case you want to extend what I said here, or in case I said something wrong.